<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorCustomCommandWidget</class>
 <widget class="QWidget" name="InspectorCustomCommandWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>235</width>
    <height>442</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutStop">
     <item>
      <spacer name="horizontalSpacerStop">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelStop">
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditStop"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPlay">
     <item>
      <spacer name="horizontalSpacerPlay">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>42</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPlay">
       <property name="text">
        <string>Play</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPlay"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLoad">
     <item>
      <spacer name="horizontalSpacerLoad">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>39</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelLoad">
       <property name="text">
        <string>Load</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditLoad"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPause">
     <item>
      <spacer name="horizontalSpacerPause">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>33</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPause">
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPause"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutNext">
     <item>
      <spacer name="horizontalSpacerNext">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>39</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelNext">
       <property name="text">
        <string>Next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditNext"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutUpdate">
     <item>
      <spacer name="horizontalSpacerUpdate">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>27</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelUpdate">
       <property name="text">
        <string>Update</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditUpdate"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutInvoke">
     <item>
      <spacer name="horizontalSpacerInvoke">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelInvoke">
       <property name="text">
        <string>Invoke</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditInvoke"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutPreview">
     <item>
      <spacer name="horizontalSpacerPreview">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>22</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPrev">
       <property name="text">
        <string>Preview</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditPreview"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutClear">
     <item>
      <spacer name="horizontalSpacerClear">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>37</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelClear">
       <property name="text">
        <string>Clear</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditClear"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutClearVideolayer">
     <item>
      <spacer name="horizontalSpacerClearVideolayer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelClearVideolayer">
       <property name="text">
        <string>Clear Videolayer</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditClearVideolayer"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutClearChannel">
     <item>
      <spacer name="horizontalSpacerClearChannel">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>15</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelClearChannel">
       <property name="text">
        <string>Clear Channel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEditClearChannel"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerFileRecorder">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEditStop</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>stopCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>155</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPlay</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>playCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>190</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>48</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditLoad</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>loadCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>204</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPause</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>pauseCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>156</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>103</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditNext</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>nextCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>200</x>
     <y>131</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditUpdate</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>updateCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>211</x>
     <y>156</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>153</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditInvoke</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>invokeCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>230</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditClear</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>clearCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>253</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>217</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditClearVideolayer</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>clearVideolayerCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditClearChannel</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>clearChannelCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>309</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>234</x>
     <y>310</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEditPreview</sender>
   <signal>textChanged(QString)</signal>
   <receiver>InspectorCustomCommandWidget</receiver>
   <slot>previewCommandChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>215</y>
    </hint>
    <hint type="destinationlabel">
     <x>9</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>stopCommandChanged(QString)</slot>
  <slot>playCommandChanged(QString)</slot>
  <slot>loadCommandChanged(QString)</slot>
  <slot>pauseCommandChanged(QString)</slot>
  <slot>nextCommandChanged(QString)</slot>
  <slot>updateCommandChanged(QString)</slot>
  <slot>invokeCommandChanged(QString)</slot>
  <slot>clearCommandChanged(QString)</slot>
  <slot>clearVideolayerCommandChanged(QString)</slot>
  <slot>clearChannelCommandChanged(QString)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>previewCommandChanged(QString)</slot>
 </slots>
</ui>
